bear/main.py.in

```diff
@@ -435,6 +435,8 @@ def split_command(command):
                 next(args)
         elif re.match(r'^-(l|L|Wl,).+', arg):
             pass
+        elif re.match(r'^-Wp,-M.+', arg):
```

Comment:
I don't think it can be put in IGNORED_FLAGS as it is a regexp, or did I miss something?

The actual flags passed looks like this: `gcc -Wp,-MP -Wp,-MMD,.depend-user_errors.o -Wp,-MT,user/errors.o ...`. It is used by our internal makefiles at work, which for some reason (probably because the preprocessor grew the features before they were exposed in gcc driver) pass the flags preprocessor using `-Wp,-Mxxx` options instead of just passing `-Mxxx` to gcc and let it pass it on to the preprocessor.